wdio/v7/typescript-basic/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "lib": ["es2016", "dom"],
+    "types": ["webdriverio/async"]
```

Comment:
Why is this needed?

Reply:
This resolves a type error when viewing the types from WDIO v7
<img width="815" alt="image" src="https://github.com/dequelabs/watcher-examples/assets/41127686/bf3feea2-9475-4d28-8399-aa342e63d584">


it doesn't affect the types within the tests but to resolve the type inspection, thought I'd add it: 

```
NOTE: If you hit "error TS2694: Namespace 'global.WebdriverIO' has no exported member 'Browser'" when using typescript,
 * add "webdriverio/async" into tsconfig.json's "types" array will solve it: <code> { "compilerOptions": { "types": ["webdriverio/async"] } } </code>
```